common/src/main/java/com/wynntils/utils/type/CappedValue.java

```diff
@@ -7,12 +7,41 @@
 public record CappedValue(int current, int max) {
     public static final CappedValue EMPTY = new CappedValue(0, 0);
 
-    public int getPercentage() {
-        return Math.round((float) current / max * 100.0f);
+    public CappedValue withCurrent(int newCurrent) {
+        return new CappedValue(newCurrent, max);
+    }
+
+    /** Return true iff the current value is equal to the max value */
```

Comment:
```suggestion
    /** Return true if the current value is equal to the max value */
```

Reply:
Actually, that is not a typo. It's a shorthand for "if and only if". https://en.wikipedia.org/wiki/If_and_only_if

Reply:
Never seen that :) Good to know.